tests/k6/tests/enduser/dialogSystemLabelLog.js

```
@@ -0,0 +1,60 @@
+        console.log(response.json());
+        expect(response.json(), 'response body').to.have.lengthOf(1);
+    });
+
+    describe('Changing labels trigger 2 logs', () => {
+
+        let body = {
+            'label': 'archive'
+        }
+        let response = putEU('dialogs/' + dialogId + '/systemlabels', body);
+        expectStatusFor(response).to.equal(204);
+        response = getEU('dialogs/' + dialogId + '/labellog');
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)